nautobot/docs/additional-features/jobs.md

```diff
@@ -98,6 +98,14 @@ class MyJob(Job):
 
 A list of strings (field names) representing the order your form fields should appear. If not defined, fields will appear in order of their definition in the code.
 
+#### `hidden`
+
+Default: `False`
+
+A Boolean that if set to `True` will prevent the job from being displayed in the list of jobs in the Nautobot UI.
```

Comment:
Maybe elaborate a bit more on what this means in practice, i.e.:

- Not shown in the base "Jobs List" page, however...
- Still runnable through the REST API
- Still runnable through the UI if a user knows or can guess the correct URL for the job submission form
- Results for this job still appear in the Job Results list after it is run

Basically emphasize that this is not a security feature, it's a UI "cleanliness" feature.